hologram_dashboard/extension/src/app/dashboard/dashboard.component.ts

```diff
@@ -22,40 +26,41 @@ export interface HologramDataAvailability {
 })
 export class DashboardComponent implements OnInit {
   dashboard: Array<HologramDataAvailability>;
-  showChipper: boolean
-  showChipperGDPR: boolean
+  showChipper: boolean;
+  showChipperGDPR: boolean;
 
-  constructor(private requestHandler: RequestHandler) { 
+  constructor(private requestHandler: RequestHandler) {
     this.showChipper = false;
     this.showChipperGDPR = false;
-    this.dashboard = []
+    this.dashboard = [];
   }
 
   ngOnInit(): void {
   }
 
-  renderTemplate(hologram_data_availability: JSON, system: string) {
-    for (let key in hologram_data_availability) {
-      let data_set_availability = {} as HologramDataAvailability;
-      data_set_availability.sourceType = key;
-      // Need to initialize the list before pushing into it;
-      data_set_availability.dateAvailabilityList = [];
-      hologram_data_availability[key].forEach(data => {
-        let date_info = {} as DateAvailability;
-        // data is a list of format [date, availability]
-        date_info.date = data[0];
-        date_info.isAvailable = data[1];
-        data_set_availability.dateAvailabilityList.push(date_info);
+  renderTemplate(inputData: JSON) {
+    for (let key of Object.keys(inputData)) {
+      let hologramDataAvailability = {} as HologramDataAvailability;
+      hologramDataAvailability.sourceType = key;
+      /**Need to initialize the list before pushing into it. */
```

Comment:
nit: //
/** .. */ is the typescript doc, on the class level. For regular comments, just use //